evennia/commands/default/admin.py

```diff
@@ -483,8 +481,7 @@ def func(self):
         locktype = "edit" if accountmode else "control"
         if not obj.access(caller, locktype):
             caller.msg(
-                "You are not allowed to edit this %s's permissions."
-                % ("account" if accountmode else "object")
+                f"You are not allowed to edit this {'account' if accountmode else 'object'}'s permissions."
```

Comment:
While this works, it may be better to use `.format()` here, or break out the code. It's generally a bit harder to read full executable code inside a string like this. I would suggest this instead: 

```python
    accountstr = 'account' if accountmode else 'object'
    caller.msg(f"You are not allowed to edit this {accountstr}'s permissions")
```